allvers/2007-07-16-GPL-2.0-from-gnu.org.txt.notes

```diff
@@ -0,0 +1,8 @@
+Add sixth version of the GPL 2 aka, the "graveyard" version
+
+* shortly after the publication of the GPL 3.0, the good ole GPL 2.0
+  was retired to the old licenses cemetery. And along the GPL 2 MK5
+  received a last, final update with extra lesser vs library changes and
+  formatting fixes
```

Comment:
I actually do not see the "lesser vs library" changes compared to 2006-07-01 (GNU)

Reply:
Fixed in https://github.com/pombredanne/gpl-history/pull/18/commits/ff621c5271f642fc82151b2cf91bd7ce420d589f